test/redis/integration.js

```diff
@@ -239,4 +239,41 @@ describe('redis integration:', () => {
       assert.isNull(await redis.get('foo'));
     });
   });
+
+  describe('zadd:', () => {
```

Comment:
You add support for the following methods:

>   'zadd', 'zcard', 'zcount', 'zpopmax', 'zpopmin', 'zrange', 'zrangebyscore', 'zrank', 'zrem',
  'zremrangebyrank', 'zremrangebyscore', 'zrevrange', 'zrevrangebyscore', 'zrevrank', 'zscore',

Should we assert all of these work as expected?

Reply:
I've removed the ones that weren't being tested.

Reply:
Note I'm using `zrem` in the verification reminders code now, so will add that and an appropriate test to this PR.